docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -19,6 +19,34 @@ This is a list of things to check if you want to update from a previous version
 - Optionally check also accessory files like `.eslinrc`, if you want to keep aligned with lint standards.
 - Follow the instructions below, in order, from your version to the one you want to update to.
 
+## Migration from 2020.02.00 to 2020.03.00
+
+### App structure review
+
```

Comment:
I move this on top because it is probably the only one that have to be effectively changed
```suggestion
From this version some base components of MapStore App has been restructured and organized.

- `web/client/product/main.jsx` has been updated to new `import` and `export` syntax (removed `require` and `exports.module`). So if you are importing it (usually in your `app.jsx`) you have to use the `import` syntax or use `require(...).default` in your project.
```